gtsam/navigation/CombinedImuFactor.h

```diff
@@ -171,6 +164,15 @@ class GTSAM_EXPORT PreintegratedCombinedMeasurementsT : public PreintegrationTyp
 
   /// @}
 
+#ifdef GTSAM_ALLOW_DEPRECATED_SINCE_V43
+/// @deprecated: biasAccOmegaInt is no longer used. Use a prior on first bias instead.
+  void resetIntegration(const gtsam::Matrix6& Q_init) {
```

Comment:
We should have a new version that is something like:
```cpp
void resetIntegration() {
    PreintegrationType::resetIntegration();
    preintMeasCov_.setZero();
}
```

Reply:
It is there.